data/apis/finage/apiMetadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Finage",
+  "active": false,
+  "description": "Finage is a financial data vendor company. We are building our services with the latest technologies and serving uninterrupted and the fastest data possible. Trusted by 18,000+ developers, quants, businesses, universities, forex firms, cryptocurrency trade apps, and individuals.",
```

Comment:
```suggestion
  "description": "Finage is a financial data vendor company. We have built our services with the latest technologies to provide uninterrupted data with the lowest delay possible.",
```

Reply:
This is probably copy pasted from somewhere and we should ask for them when we use them. Do we use them atm?

Reply:
It's from their website, but yeah we should ask them. 
i guess same goes for Amberdata.